test/test_cli/test_deterministic_smac.py

```diff
@@ -71,8 +76,8 @@ def test_deterministic(self, patch):
         h1 = json.load(open(self.output_dir_1 + '/run_1/runhistory.json'))
         h2 = json.load(open(self.output_dir_2 + '/run_1/runhistory.json'))
         h3 = json.load(open(self.output_dir_3 + '/run_2/runhistory.json'))
-        self.assertEqual(h1, h2)
-        self.assertNotEqual(h1, h3)
+        self.assertEqual(self.ignore_timestamps(h1), self.ignore_timestamps(h2))
+        self.assertNotEqual(self.ignore_timestamps(h1), self.ignore_timestamps(h3))
```

Comment:
h1 will be changed in l80, so this check will always pass

Reply:
Fixed, please have a look.